support-e2e/tests/tieredCheckout.test.ts

```diff
@@ -0,0 +1,104 @@
+import { expect, test } from '@playwright/test';
+import { email, firstName, lastName } from './utils/users';
+import { setTestUserDetails } from './utils/testUserDetails';
+import { checkRecaptcha } from './utils/recaptcha';
+import { fillInCardDetails } from './utils/cardDetails';
+import { fillInDirectDebitDetails } from './utils/directDebitDetails';
+import { fillInPayPalDetails } from './utils/paypal';
+import { setupPage } from './utils/page';
+import { afterEachTasks } from './utils/afterEachTest';
+
+interface TestDetails {
+	paymentType: 'Credit/Debit card' | 'Direct debit' | 'PayPal';
+	tier: 1 | 2 | 3;
+	frequency: 'Monthly' | 'Annual';
+	country?: 'US' | 'AU';
```

Comment:
Worth adding a European country or `EU`?

Reply:
I don't think so, I think the only reason we test a US variant is because there are a couple additional fields on the US checkout compared to UK. We don't actually have any tests which test AU (but this was there in the original `recurringContributions.tests.ts` file so I've left it here - there'd be no benefit in testing AU as well as UK as there's no extra functionality.